wsgi/odyssy/odyssy/settings.py

```diff
@@ -57,6 +57,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'announcement'
```

Comment:
:+1:  Done. But what is the difference can u explain? Just for knowledge purpose.